Gemfile.lock

```diff
@@ -846,6 +844,7 @@ DEPENDENCIES
   uglifier (>= 1.0.3)
   unicorn
   unicorn-rails
+  web!
```

Comment:
Bundler, why are u doing this?

![](https://media.giphy.com/media/njl60xjwgrkWY/giphy.gif)

Reply:
I explain this here:
https://github.com/openfoodfoundation/openfoodnetwork/pull/2735#issuecomment-431141910

Reply:
it's putting it in the right place (alphabetical order), I edited the gemfile.lock myself (renamed the engine)! that's what it is correcting.

Reply:
so the move is from the alphabetical order position of "content" (previous name) to "web" (new name)